src/unix/linux_like/linux/mod.rs

```diff
@@ -2417,6 +2432,56 @@ pub const IN_ALL_EVENTS: u32 = IN_ACCESS
 pub const IN_CLOEXEC: ::c_int = O_CLOEXEC;
 pub const IN_NONBLOCK: ::c_int = O_NONBLOCK;
 
+// uapi/linux/fanotify.h
+pub const FAN_ACCESS: ::c_ulonglong = 0x0000_0001;
+pub const FAN_MODIFY: ::c_ulonglong = 0x0000_0002;
+pub const FAN_CLOSE_WRITE: ::c_ulonglong = 0x0000_0008;
+pub const FAN_CLOSE_NOWRITE: ::c_ulonglong = 0x0000_0010;
+pub const FAN_OPEN: ::c_ulonglong = 0x0000_0020;
+
+pub const FAN_Q_OVERFLOW: ::c_ulonglong = 0x0000_4000;
+
+pub const FAN_OPEN_PERM: ::c_ulonglong = 0x0001_0000;
+pub const FAN_ACCESS_PERM: ::c_ulonglong = 0x0002_0000;
+
+pub const FAN_ONDIR: ::c_ulonglong = 0x4000_0000;
+
+pub const FAN_EVENT_ON_CHILD: ::c_ulonglong = 0x0800_0000;
+
+pub const FAN_CLOSE: ::c_ulonglong = FAN_CLOSE_WRITE | FAN_CLOSE_NOWRITE;
```

Comment:
Should they really be `c_ulonglong`?

Reply:
I changed these to `u64` in 42ac577 to match the `mask` argument to `fanotify_mark`: `uint64_t mask` in C and `mask: u64` in this branch as of https://github.com/rust-lang/libc/pull/1699/commits/f082b9935022a7635b1a78ddc16bac2cc01566d8